audiobook/main.py

```diff
@@ -45,15 +63,28 @@ def create_json_book(self, pdf_file_path, password=None):
                 book_dict[num] = text
         return book_dict, pages
     
-    def save_audio(self, pdf_file_path, password=None):
-        if not os.path.exists(pdf_file_path):
-            raise FileNotFoundError("File not found!")
+    def txt_to_json(self, input_file_path):
+        book_dict = {}
+        with open(input_file_path, "r") as fp:
+            file_data = fp.read()
         
-        if not pdf_file_path.endswith(".pdf"):
-            raise ValueError("File must be a pdf!")
+        # split text into pages of 2000 characters
+        for i in range(0, len(file_data), 2000):
+            book_dict[i] = file_data[i:i+2000]
+        return book_dict, len(book_dict)
         
-        with open(pdf_file_path, "rb") as fp:
-            basename = os.path.basename(pdf_file_path).split(".")[0]
+    def create_json_book(self, input_file_path, password=None):
+        self.file_check(input_file_path)
+        if input_file_path.endswith(".pdf"):
+            book_dict, pages = self.pdf_to_json(input_file_path, password)
+        elif input_file_path.endswith(".txt"):
+            book_dict, pages = self.txt_to_json(input_file_path)
+        return book_dict, pages
+    
+    def save_audio(self, input_file_path, password=None):
+        self.file_check(input_file_path)
+        with open(input_file_path, "rb") as fp:
+            basename = os.path.basename(input_file_path).split(".")[0]
             os.makedirs(basename, exist_ok=True)
```

Comment:
you forget to get book_dict in save_audio function.

Reply:
Thanks for pointing out the issue